docs/onboarding.md

```diff
@@ -43,3 +43,18 @@ Then add your SSH key _also_ as the commit signing key:
 1. Open [New SSH Key](https://github.com/settings/ssh/new) page.
 2. Select `Key type`: `Signing key`.
 3. Copy the content of `~/.ssh/id_rsa.pub` file (or other file you used in `user.signingKey` above).
+
+If you're on WSL2, this may help:
+
+- Add those lines to `~/.gnupg/gpg.conf`
+
+```sh
```

Comment:
Increase indent of code block. Now Markdown parser will separated `<ul>` since it don’t know that code block is inside the list.